### PR TITLE
Fixes for issue #14 and have added some more explicit package versions

### PR DESCRIPTION
**Pull request by JohnRoach**
This pull request is for issue #14

### CODE REVIEW

File: shell.py [hunk 1]
```diff
@@ -35,18 +48,16 @@ def show_help_text(self, keyword):
...
     keywords = ['commands']
-    help_text = """{keyword}
-{divider}
-Summary: List available commands
-
-Usage: {keyword}
-"""
+    help_text = "{keyword}\n" \
+                "{divider}\n" \
+                "Summary: List available commands\n" \
```
Comment (mpirnat) [inline shell.py]: I am highly allergic to explicit \n in blocks of text, so I'd like to figure out something better for this. (I admit that having to dedent is gross too.)

File: shell.py [hunk 2]
```diff
@@ -245,12 +251,19 @@ class NextTurn(Command):
 
     keywords = ['next']
 
+    @manager.registry.add_binding(Keys.ControlN)
     def do_command(self, *args):
-        if not self.game.tm:
+        if not game.tm:
```
Comment (mpirnat) [inline shell.py]: How the heck does this even work without the self?? This feels way too magical.

File: shell.py [hunk 3]
```diff
@@ -245,12 +251,19 @@ class NextTurn(Command):
...
+            if not hasattr(self, "game"):
+            # this feels very hacky however it looks like key binding
+            # takes over object. To me it looks like we might want to change
+            # approach on how we load commands.
```
Comment (mpirnat) [inline shell.py]: Agreed, this is super weird. Also I was never in love with the command registration setup anyway, so I am definitely open to fixing that up at some point!

File: shell.py (hunk 1 above)
Comment (JohnRoach) [reply shell.py]: no problem I will revert it. I just found the dedent more annoying than this 😆

File: shell.py (hunk 2 above)
Comment (JohnRoach) [reply shell.py]: Python :roll_eyes: haha 😆

File: shell.py (hunk 3 above)
Comment (JohnRoach) [reply shell.py]: sure thing.

**Pull request merged**